### PR TITLE
Use new segmented control in viewer

### LINKED ISSUE #6147: Tracking downloads by Segment
# What is it and who's it for?
Tracking when users download items for feature evaluation

# User story
As an analyst

I want to track the conversion rate of total downloads/total item views and identify API/workType where users are getting stuck

So that the Experience team can improve access

# Implementation 

## Acceptance criteria
Field available in conversion* index

## Testing and tracking
To be built into conversion dashboard

## Other considerations

### PR DESCRIPTION
**Pull request by davidpmccormick**
Closes #6147 

## Who is this for?
Users of of the item viewer

## What is it doing for them?
Giving them a new control for toggling between detail/grid view that is more consistent with the rest of the ui.

image

The intention is to a/b test the presence/absence of labels alongside the icons in the control, but we decided to wait until the a/b test for label presence on the sidebar toggle was complete.

### CODE REVIEW

File: common/views/components/IIIFViewerPrototype/ViewerTopBarPrototype.tsx [hunk 1]
```diff
@@ -228,26 +226,38 @@ const ViewerTopBar: FunctionComponent<Props> = ({
...
-              <Icon name={gridVisible ? 'detailView' : 'gridView'} />
-              <span className={`btn__text`}>
-                {gridVisible ? 'Detail view' : 'View all'}
-              </span>
-            </ShameButton>
+            <ToolbarSegmentedControl
+              hideLabels={true}
+              items={[
+                {
+                  id: '1',
```
Comment (jamesgorrie) [inline common/views/components/IIIFViewerPrototype/ViewerTopBarPrototype.tsx:233]: Worth using something more descriptive e.g. `page` & `grid` for ease of reading in the conditions?

**Review by jamesgorrie — APPROVED** [review]

File: common/views/components/IIIFViewerPrototype/ViewerTopBarPrototype.tsx (hunk 1 above)
Comment (jamieparkinson) [inline common/views/components/IIIFViewerPrototype/ViewerTopBarPrototype.tsx:233]: Is there a reason we can't use descriptive IDs (`detailView`/`gridView` etc) here?

**Review by jamieparkinson — APPROVED** [review]

File: common/views/components/IIIFViewerPrototype/ViewerTopBarPrototype.tsx (hunk 1 above)
Comment (jamieparkinson) [reply common/views/components/IIIFViewerPrototype/ViewerTopBarPrototype.tsx:233]: Ha snap @jamesgorrie

Comment (davidpmccormick) [reply common/views/components/IIIFViewerPrototype/ViewerTopBarPrototype.tsx:233]: there isn't, nope. Will update

**Pull request merged**